src/components/SearchBar.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react'
+
+export default class SearchBar extends Component {
+
+
+    render() {
+        return (
+            <div>
```

Comment:
I'd recommend changing this to form and adding an onSubmit to it. That way the button will actually function.